pyproject.toml

```diff
@@ -101,9 +100,11 @@ jinja2 = ">=3.0.0"
 netaddr = ">=0.7.0"
 matrix-common = "^1.1.0"
 phonenumbers = ">=8.12.32"
+prometheus-client = { version = "*", optional = true }
```

Comment:
slightly wonder if we should copy the requirement from either Synapse or Sygnal rather than go wild

Reply:
I'll change this to match Synapse, and use `prometheus_client >=0.4.0` and  `sentry-sdk >=0.7.2`.

(One could argue this ought to be semver-pinned rather than just use a lower bound. But the lockfile helps in this area to some extent)